contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -48,6 +53,9 @@ new Vue ({
   }
 });
 ```
+<box type="warning">
+Note that `{\{` and `}\}` should not have the slash in your actual code.
```

Comment:
Add a blank line before this line so that MarkBind parses it as Markdown text.
![image](https://user-images.githubusercontent.com/1673303/56075662-bca95f80-5df8-11e9-98fc-526b2625295d.png)